evofinder.py

```diff
@@ -268,6 +267,7 @@ def start_evolution(ast, target_line, baseline_file_list=[]):
   input_data = []
   best_input = ""
   best_fitness = 0
+  optimal_fitness = len(target_location_id.split("_"))
```

Comment:
You actually need to subtract one from this because the IDs always start with a '_' so you get an empty string as the first result of the split.